client/extensions/zoninator/components/forms/zone-details-form/index.jsx

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { reduxForm } from 'redux-form';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card';
+import FormButton from 'components/forms/form-button';
+import FormTextInput from 'components/forms/form-text-input';
+import FormTextarea from 'components/forms/form-textarea';
+import ReduxFormFieldset from 'components/redux-forms/redux-form-fieldset';
+import SectionHeader from 'components/section-header';
+
+const form = 'extensions.zoninator.zoneDetails';
+
+const ZoneDetailsForm = ( {
+	handleSubmit,
+	label,
+	onSubmit,
+	submitting,
+} ) => {
+	const save = data => onSubmit( form, data );
```

Comment:
May want to consider making `ZoneDetailsForm` extend `Component` since it contains a function in order to improve performance.